contributors/guide/README.md

```diff
@@ -168,9 +168,17 @@ _Improvements needed_
 
 ## Testing
 
-_Improvements needed_
-* link to testing process 
-* walkthrough of where to find what in the tests (how to use, how to debug)
+Testing is the responsibility of all contributors and is in part owned by all sigs, but is also coordinated by [sig-testing](https://github.com/kubernetes/community/tree/master/sig-testing).
+
+The main testing overview document is [here](https://github.com/kubernetes/community/blob/master/contributors/devel/testing.md).
```

Comment:
Link should be: `/contributors/devel/testing.md`

Reply:
fixed